libs/ml/tests/ml/ops/leaky_relu_op.cpp

```diff
@@ -35,21 +35,22 @@ TYPED_TEST_CASE(LeakyReluOpTest, MyTypes);
 
 TYPED_TEST(LeakyReluOpTest, forward_test)
 {
-  using DataType  = typename TypeParam::Type;
   using ArrayType = TypeParam;
 
-  ArrayType           data(8);
-  ArrayType           alpha(8);
-  ArrayType           gt(8);
-  std::vector<double> data_input({1, -2, 3, -4, 5, -6, 7, -8});
-  std::vector<double> alphaInput({0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8});
-  std::vector<double> gt_input({1, -0.4, 3, -1.6, 5, -3.6, 7, -6.4});
-  for (std::uint64_t i(0); i < 8; ++i)
-  {
-    data.Set(i, DataType(data_input[i]));
-    gt.Set(i, DataType(gt_input[i]));
-    alpha.Set(i, DataType(alphaInput[i]));
-  }
+  ArrayType data = ArrayType::FromString(R"(
```

Comment:
no need for raw string literals btw

Reply:
But without that I can't have FromString nicely on multiple lines.

Reply:
But without that I cannot have FromString on multiple lines.